include/zipped_rdd.h

```diff
@@ -0,0 +1,64 @@
+#ifndef CPARK_ZIPPED_RDD_H
+#define CPARK_ZIPPED_RDD_H
+
+#include <iterator>
+#include <ranges>
+#include <vector>
+
+#include "base_rdd.h"
+#include "utils.h"
+
+namespace cpark {
+
+/**
+ * An Rdd holding the zipped data from two old rdds
+ * @tparam R1 Type of the first old Rdd.
+ * @tparam R2 Type of the second old Rdd.
+ */
+template <concepts::Rdd R1, concepts::Rdd R2>
+class ZippedRdd : public BaseRdd<ZippedRdd<R1, R2>> {
+public:
+  using Base = BaseRdd<ZippedRdd<R1, R2>>;
+  friend Base;
+
+  using V1 = utils::RddElementType<R1>;
+  using V2 = utils::RddElementType<R2>;
+
+  constexpr ZippedRdd(const R1& prev1, const R2& prev2) : Base{prev1, false} {
```

Comment:
Shall we unify the name convention? Currently, it looks like some Rdd's are named in the present tense (Sample, Merge), and some are named in the past tense (Zipped, Transformed).

Reply:
I think it is a relatively minor issue. But if any of you are available, it does not hurt to rename them to past tense.